pom.xml

```diff
@@ -3,86 +3,114 @@
          xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>com.phonepe</groupId>
```

Comment:
revert all phonepe specific changes

Reply:
added this to deploy on phonepe artifactory for testing purpose, will remove once end-to-end testing is done